docs/website/blog/2023-05-15-hacker-news-gpt-4-dashboard-demo.md

```diff
@@ -0,0 +1,31 @@
+# Understanding how developers view ELT tools using the Hacker News API and GPT-4
```

Comment:
@rahuljo Take a look at the other blog posts. You should also add a header here (e.g. your photo and details should show up here)